app/helpers/admin/users_helper.rb

```diff
@@ -58,6 +58,10 @@ def user_status(user)
       end
     end
 
+    def filterable_statuses
+      Constants::Role::BASE_ROLES.reject { |status| status == "Good standing" }
```

Comment:
Do you think we should add a comment about why we're not adding "Good Standing" here just for our future selves or other folks that may wonder in the future why we're treating it as an exception?

Reply:
Great idea - I've added one now 🙂